lib/aws-embedded-metrics-customink/sinks/logger.rb

```diff
@@ -0,0 +1,22 @@
+module Aws
+  module Embedded
+    module Metrics
+      module Sinks
+        class Logger
+
+          attr_reader :logger, :level
+
+          def initialize(logger, level: 'info')
```

Comment:
Let's default this to a symbol

```suggestion
          def initialize(logger, level: :info)
```